package.json

```diff
@@ -5,6 +5,7 @@
     "test": "gulp test",
     "clean": "gulp clean",
     "start": "cross-env NODE_ENV=production gulp build+start --obj-dir-name production",
+    "serve": "cross-env NODE_ENV=production gulp build+serve",
     "dev": "cross-env NODE_ENV=development gulp build+start --obj-dir-name development",
     "dev+logging": "cross-env NODE_ENV=development LOGGING=on gulp build+start --obj-dir-name development",
```

Comment:
I'd rename the obj dirnames for these two "development-electron", to make it more obvious that we're building with an electron platform. Followup material.